docs/pages/api-docs/badge.md

```diff
@@ -36,7 +36,7 @@ The `MuiBadge` name can be used for providing [default props](/customization/glo
 | <span class="prop-name">component</span> | <span class="prop-type">elementType</span> | <span class="prop-default">'span'</span> | The component used for the root node. Either a string to use a HTML element or a component. |
 | <span class="prop-name">invisible</span> | <span class="prop-type">bool</span> |  | If `true`, the badge will be invisible. |
 | <span class="prop-name">max</span> | <span class="prop-type">number</span> | <span class="prop-default">99</span> | Max count to show. |
-| <span class="prop-name">overlap</span> | <span class="prop-type">'circle'<br>&#124;&nbsp;'rectangle'</span> | <span class="prop-default">'rectangle'</span> | Wrapped shape the badge should overlap. |
+| <span class="prop-name">overlap</span> | <span class="prop-type">'circle'<br>&#124;&nbsp;'rectangle'<br>&#124;&nbsp;'circular'<br>&#124;&nbsp;'rectangular'</span> | <span class="prop-default">'rectangle'</span> | Wrapped shape the badge should overlap. |
```

Comment:
That would be a breaking change. Let's say you override the `*Rectangle*` class. Switching the default to `rectangular` would ignore these overrides.

But I think I need to take a look at the docs. They might be riddled with deprecation warnings and it might be good to adjust the theme so that it's "ready" for v5. Might even be a good idea to have a "v5 theme" that smoothes the migration path.

Reply:
Docs are fine since this uses the original commit. Still, docs are now using the new default values. Might make sense to extract this into a new theme creator for migration